RealmSwift/Tests/TestCase.swift

```diff
@@ -53,6 +53,10 @@ class TestCase: RLMAutoreleasePoolTestCase {
         RLMAssertThrows(self, { _ = block() } as dispatch_block_t, message, fileName, lineNumber);
     }
 
+    func assertNil<T>(@autoclosure block: () -> T?, _ message: String? = nil, fileName: String = __FILE__, lineNumber: UInt = __LINE__) {
```

Comment:
`XCTAssertNil` requires the expression be of type `AnyObject?`. `UInt64?` isn't.